charts/internal/e2e-resources/Chart.yaml

```diff
@@ -3,6 +3,11 @@ version: 1.3.0-devel
 description: This chart creates e2e resources for nri-kubernetes.
 name: e2e-resources
 
+dependencies:
+  - name: kube-state-metrics
+    version: 4.23.0
```

Comment:
So when installing the chart we are using the `--set kube-state-metrics.image.tag` flag to choose the correct dependency version for `kube-state-metrics`

Is that flag enough to override this dependency declaration? (I'm new to Helm, so feel free to ignore comment if it does not help)

Reply:
Hi Juan, good question. Just want to clarify, we are still using the same KSM Helm chart version (“4.23”), but we set to use different KSM image versions (aka application version). To my knowledge, the KSM image is the actually KSM application whereas KSM helm chart contains dependency info so that the KSM image can work without any problems.